internal/pkg/deploy/cloudformation/cloudformation_integration_test.go

```diff
@@ -55,6 +55,7 @@ func Test_App_Infrastructure(t *testing.T) {
 	deployer := cloudformation.New(sess, cloudformation.WithProgressTracker(os.Stderr))
 	cfClient := awsCF.New(sess)
 	require.NoError(t, err)
+	version.Version = "v1.28.0"
```

Comment:
what about we just nuke this file 🤔

Reply:
Yeah I am inclined too but I feel like we need at least a separate PR to remove the integration test.